swiftsimio/metadata/particle/particle_types.py

```diff
@@ -4,31 +4,31 @@
 
 # Describes the conversion of particle types to names
 particle_name_underscores = {
-    0: "gas",
-    1: "dark_matter",
-    2: "boundary",
-    3: "sinks",
-    4: "stars",
-    5: "black_holes",
-    6: "neutrinos",
+    "PartType0": "gas",
+    "PartType1": "dark_matter",
+    "PartType2": "boundary",
+    "PartType3": "sinks",
+    "PartType4": "stars",
+    "PartType5": "black_holes",
+    "PartType6": "neutrinos",
 }
 
 particle_name_class = {
-    0: "Gas",
-    1: "DarkMatter",
-    2: "Boundary",
-    3: "Sinks",
-    4: "Stars",
-    5: "BlackHoles",
-    6: "Neutrinos",
+    "PartType0": "Gas",
+    "PartType1": "DarkMatter",
+    "PartType2": "Boundary",
+    "PartType3": "Sinks",
+    "PartType4": "Stars",
+    "PartType5": "BlackHoles",
+    "PartType6": "Neutrinos",
 }
 
 particle_name_text = {
-    0: "Gas",
-    1: "Dark Matter",
-    2: "Boundary",
-    3: "Sinks",
-    4: "Stars",
-    5: "Black Holes",
-    6: "Neutrinos",
+    "PartType0": "Gas",
+    "PartType1": "Dark Matter",
+    "PartType2": "Boundary",
+    "PartType3": "Sinks",
+    "PartType4": "Stars",
+    "PartType5": "Black Holes",
+    "PartType6": "Neutrinos",
```

Comment:
Why did you change this?

Reply:
I changed `present_particle_types` to return "PartType{i}" rather than integers
https://github.com/SWIFTSIM/swiftsimio/blob/143d65ff3b68cf85f6aba914885b0e96295ed769/swiftsimio/reader.py#L652
So I could either change this file, or else I'd have to convert back to an integer when getting the particle name

Reply:
Are we not able to remove these for modern snapshots? As they contain the particle type names and fields named appropriately..? I'd like to use those so we don't have to add/handle additional or renamed particle types manually.